ui/src/app/components/apps-panel/expanded-app.tsx

```diff
@@ -293,10 +303,14 @@ export const ExpandedApp = (props: ExpandedAppProps) => {
       ) : (
         <FlexColumn>
           {/* TODO: keep status updated internally */}
```

Comment:
Is this TODO still relevant?

Reply:
I do not believe so. Thanks for pointing it out.

Reply:
nope: this polling is what the TODO is for.  Thanks!